### DISCUSSION TITLE
Установка на локальную машину

### DISCUSSION THREAD

**Discussion by rusovich-sergey** [totumonline/totum-ru-issues-and-discussions#365] [Как это сделать?] [answered]
Хочется хорошенmко протестировать вашу систему, но не имею хостинга, можно ли установить все это дело на виртуалку?  Или ещё лучше под виндой ? Если да то может есть какая нибудь инструкция ? Спасибо.

**Comment by LostAlly** [comment, marked as answer]
1. Конечно можно поставить на виртуалу бунту 20.4 и на нее тотем
2. В винде есть встроенная виртуалищация, из магазина ставите убунту, и на нее тотем, но так как это уже не совсем естественная среда для тотума, могут быть вопросы с установкой если вы с линуксом не на ты, я бы не связывался.

**Reply by ttm-alexeypolunin** [reply]
Дополню: вот этот скрипт используйте для установки без домена — https://ru.docs.totum.online/ubuntu#without-domain

На шаге указания внешнего ip возможно нужно будет указать localhost, вместо определившегося виртуального ip. Вот по этой теме: https://github.com/totumonline/totum-ru-issues-and-discussions/discussions/287

Если попробовать — то на демо функционально все, кроме внешних коннектов: https://ru.totum.online/downloaddemo

Если хочется свою систему, которую потом можно будет перевести в рабочий вариант, то на шареде NetAngels за 10 минут поставите за 100 р (производительноть там ток плохая): https://ru.docs.totum.online/install_on_net_angels
